tests/test_file_storage.py

```diff
@@ -24,12 +25,22 @@ class TEST_CONSTANTS:
     TEST_FOLDER = os.environ.get("TEST_FOLDER")
     GCS_BUCKET = os.environ.get("GCS_BUCKET")
     TEXT_CONTENT = os.environ.get("TEXT_CONTENT")
-    FILE_STORAGE_ENV = "FILE_STORAGE"
+    FILE_STORAGE_GCS = "FILE_STORAGE_GCS"
+    FILE_STORAGE_MINIO = "FILE_STORAGE_MINIO"
+    FILE_STORAGE_LOCAL = "FILE_STORAGE_LOCAL"
 
 
 def file_storage(provider: FileStorageProvider):
```

Comment:
@gaya3-zipstack make this a pytest fixture with a valid scope that can be reused easily. You'd have to place it in a separate `conftest.py`

Reply:
Will take it up later